src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,59 @@
+package org.example.service;
+
+import java.sql.*;
+
+
+public class ConnectionService {
```

Comment:
Rearrange fields of this class.
Constants should be on top.

Reply:
fixed